### PR TITLE
Add Activity Status to Friends' List

### PR DESCRIPTION
**Pull request by sulaimonlasisi**
@danfstucky I worked on this a while back but just had time to push it out. The functionality allows a logged in user to see which of their summoner friends are currently online (green icon), recently active (yellow icon) or haven't been online for a while (grey icon).
Review and let me know what you think

### CODE REVIEW

File: app/views/friendships/_friends_list.html.erb [hunk 1]
```diff
@@ -12,8 +12,20 @@
 		<% else %>
 		  <td ><strong>Friends Since: <%= time_ago_in_words(friendship.request_responded_at).capitalize %> ago</strong></td>
 		<% end %>
-	  <td class="champ-img" rowspan="3" scope="rowgroup"><span class="imgWrap"><%= link_to image_tag("icons/AhriSquare.png"), profile_path(friendship.friend.id)  %></span>
-    </td>
+	  <td rowspan="3" scope="rowgroup">
+      <div class="champ-img">
+        <span class="imgWrap"><%= link_to image_tag("icons/AhriSquare.png"), profile_path(friendship.friend.id)  %></span>
+        <div class="status-icon">
+	        <% if friendship.friend.logged_in == true %>
```
Comment (danfstucky) [inline app/views/friendships/_friends_list.html.erb]: You don't need to compare this with the boolean.  `if friendship.friend.logged_in` will do the same thing

Comment (sulaimonlasisi) [inline app/views/friendships/_friends_list.html.erb]: Fixed.

**Comment by danfstucky** [comment]
+1

**Comment by jwiebe17** [comment]
Maybe I should create some testing for this entire project? Is there a need?

**Comment by danfstucky** [comment]
@jwiebe17 It's never a bad idea to have tests, but I wouldn't say there is a need for it here. This project is pretty much wrapping up now and I believe we were calling it quits. If you want to write some tests, I would say go for it though as it only makes the project better. For ruby, I recommend the RSpec testing framework.

**Pull request merged**